polling_stations/apps/bug_reports/templates/bug_reports/disclaimer.html

```diff
@@ -0,0 +1,5 @@
+<p><small>
+  If you provide an email address, we may contact you to
+  request more information or notify you of improvements.
+  It will not be used for email marketing purposes.
```

Comment:
I think we should have something to this effect here, but feedback on wording would be useful. Maybe @JoeMitchell has thoughts on this?

Reply:
i think it's good!